oscarlab/graphene
# [CI] Enable parallelism when building tests

## Description of the changes 

#1695 was merged, we should be able now to enable parallel builds for all tests (at least I hope...). This makes the biggest difference for LTP and signing manifests for SGX.

Closes #1687.

## How to test this PR? 

CI + verify logs on Jenkins if everything looks fine.


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/oscarlab/graphene/1703)
